tests/test_normalize_by_median.py

```diff
@@ -78,7 +93,7 @@ def test_normalize_by_median_quiet():
     shutil.copyfile(utils.get_test_data('test-abund-read-2.fa'), infile)
 
     script = 'normalize-by-median.py'
-    args = ['-C', CUTOFF, '-k', '17', '--quiet', infile]
+    args = ['-C', CUTOFF, '-k', '17', '--quiet', '-M', '1e6', infile]
```

Comment:
Because I made it so report_on_config executed again I had to increase the tablesize to stop it from warning during the quiet test. Normally, I'd then go "But it shouldn't make any noise when silenced anwyay" but these are warnings, which as I recall we wanted to ignore --quiet.